config.yaml

```diff
@@ -60,6 +60,15 @@ options:
       Synapse server name. Must be set to deploy the charm. Corresponds to the
       server_name option on Synapse configuration file and sets the
       public-facing domain of the server.
+  stats_exporter_user:
```

Comment:
Is this a synapse user? If so, will it be possible to create it on deployment? Otherwise we'll have a chicken egg situation, as I understand it

Reply:
Thanks, we discussed a little bit about it in the daily so I decided to change the approach again and now the Synapse Stats Exporter connects to the database to get metrics so there is no need of API user.